solana/Makefile

```diff
@@ -53,14 +53,14 @@ idl: anchor-build
 
 sdk: anchor-build
 	@echo "Building SDK"
-	cd .. && npm ci && npm run build:solana
+	cd .. && bun ci && bun run build:solana
 
 clean:
 	anchor clean
 	rm -rf .anchor node_modules
 
 node_modules:
-	npm install
+	bun install
```

Comment:
I understand this was just a 1-1 switch, but I feel like this should have probably been `bun ci`

Reply:
The good thing about bun is it doesn't upgrade any deps unless you actually change the package.json or tell it to. No need to worry about it in most places